test/NuGet.Core.Tests/NuGet.Protocol.Tests/AutoCompleteResourceV3Tests.cs

```diff
@@ -31,11 +32,14 @@ public async Task AutoCompleteResourceV3_IdStartsWithAsync()
             var repo = StaticHttpHandler.CreateSource(sourceName, Repository.Provider.GetCoreV3(), responses);
             var resource = await repo.GetResourceAsync<AutoCompleteResource>();
 
+            var logger = new TestLogger();
+
             // Act
-            var result = resource.IdStartsWith("newt", true, NullLogger.Instance, CancellationToken.None);
+            var result = resource.IdStartsWith("newt", true, logger, CancellationToken.None);
 
             // Assert
             Assert.Equal(10, result.Result.Count());
+            Assert.NotEqual(0, logger.Messages.Count);
```

Comment:
It's going correct direction. But we need some action which logs something into `logger.Messages`. If it's `0` then it's hard to tell if we're not logging correctly or nothing really happened. 
Please try to find scenario that logs something. Currently nothing comes into my mind that logs something here.

Reply:
@tintoy - Sorry for the delay in responding on this pull request. Looking at the code path I could not find any scenario where errors are logged to the logger. I noticed only information messages added to the logger. I am fine with the current test unless other team members have unique perspective.

Can you please modify tests in the below class file because `AutoCompleteResource` is also used in powershell commands?

For e.g. https://github.com/NuGet/NuGet.Client/blob/3d301ae9802e60f9ea9b0c3b0ed0948175459695/test/NuGet.Core.Tests/NuGet.Protocol.Tests/PowershellAutoCompleteResourceTests.cs#L98

Please rebase your branch with dev to get latest.

Reply:
No problem, sorry this one kinda dropped off my radar for a while too. Will get onto it ASAP :)

Reply:
So far, I don't see any test which actually logging anything.
We need something not 0 and content of message is what you intended to log.

Reply:
Ok, no worries; can I assume that any of the messages currently logged (based on inspection of the code) can be considered a suitable candidate?

Reply:
I don't know any message can be logged here, but if we're adding logging then better to something logged in there to test. What is your use case? I assume you already have one, then just add test for it.